test/Chainweb/Test/Mining.hs

```diff
@@ -67,8 +67,8 @@ withTestCoordiantor
     -> IO ()
 withTestCoordiantor rdb maybeConf a = do
     var <- newEmptyMVar
-    x <- race (takeMVar var) $ do
-        withTestCutDb rdb v 0 (\_ _ -> return fakePact) (logFunction logger) $ \cdb ->
+    x <- race (takeMVar var) $ 
+        fmap snd $ withTestCutDb rdb v id 0 (\_ _ -> return fakePact) (logFunction logger) $ \cdb ->
```

Comment:
see comment above on `test/Chaiwneb/Test/CutDb.hs` regarding the change of the return type of `withTestCutDb`.